### ISSUE TITLE
Scale Layer not supported

### ISSUE THREAD

**Issue by noio** [tiny-dnn/tiny-dnn#884]
First off, let me say I love tiny-dnn and it's philosophy! Went through a lot of options but tiny-dnn is definitely the easiest to hook up and very pleasant to use.

I came a long way with this network: I used half of a YOLO network (pretrained weights in darknet format). Converted that to PyTorch to finetune, then exported again using [pytorch-caffe-darknet-convert](https://github.com/marvis/pytorch-caffe-darknet-convert) to Caffe's format. 

However, this method splits PyTorch's BatchNorm layers into a Caffe BN plus a Caffe **Scale** layer. And there lies the obvious problem, since tiny-dnn doesn't support this Scale layer that ends up in the file. 

1. I'm not looking to _train_ the network in tiny-dnn, is there a way to "bake" the scaling into another layer? (Perhaps even before I export to Caffe?)
2. Are there any plans to implement a Scale layer?

If the answer to 2. is **no**, then feel free to close this issue. I still thought it would be good to open one, as others might have the same question.
